### PR TITLE
making workflow agnostic to location

### PR DESCRIPTION
**Pull request by SBFRF**
making workflow agnostic to FRF.  Using VA beach data as example. 
- exposure of new flags to process data with different thresholds
-  moved plots out of main body of workflow into `yellowfinlib`

### CODE REVIEW

**Comment by SBFRF** [comment]
@kkoetje here's where you do the reivew

File: workflow_ppk.py [hunk 1]
```diff
@@ -328,7 +288,7 @@ def main(datadir, geoid, makePos=True, verbose=2, sonar_method='instant', rtklib
     print("here's where some better filtering could be done, probably worth saving an intermediate product here "
           "for future revisit")
 
-    print(f"for now we're only saving/logging values that have a GNSS fix quality of {ppk_quality_threshold} and a "
+    logging.info(f"saving/logging values that have a GNSS fix quality of {ppk_quality_threshold} and a "
           f"sonar confidence > {smoothed_sonar_confidence}")
```
Comment (kkoetje) [inline workflow_ppk.py:292]: Add a few spaces to vertically align to left parenthesis?

File: yellowfinLib.py [hunk 2]
```diff
@@ -1099,3 +1099,149 @@ def transectSelection(data, **kwargs):
...
+
+def qaqc_time_offset_determination(ofname, pc_time_off):
+        # Compare GPS data to make sure timing is ok
+        plt.figure()
+        plt.suptitle('time offset between pc time and GPS time')
+        ax1 = plt.subplot(121)
+        ax1.plot(pc_time_off, '.')
+        ax1.set_xlabel('PC time')
+        ax1.set_ylabel('PC time - GGA string time (+leap seconds)')
+        ax2 = plt.subplot(122)
```
Comment (kkoetje) [inline yellowfinLib.py:1190]: Extra tab/spaces inside this function?

**Review by kkoetje — APPROVED** [review]
A few small formatting comments to look at. In yellowfinLib.py the h5 file needs to be closed before line 339. Will make this change in a future commit.

**Comment by SBFRF** [comment]
> A few small formatting comments to look at. In yellowfinLib.py the h5 file needs to be closed before line 339. Will make this change in a future commit.

I fixed this in this latest PR.

**Pull request merged**